freeipa/src/main/java/com/sequenceiq/freeipa/service/upgrade/UpgradeService.java

```diff
@@ -130,4 +138,22 @@ private List<ImageInfoResponse> getTargetImages(String catalog, Stack stack, Ima
         LOGGER.debug("Found target images: {}", targetImages);
         return targetImages;
     }
+
+    String calculateUpgradeVariant(Stack stack, String accountId) {
+        String variant = stack.getPlatformvariant();
+        boolean migrationEnable = entitlementService.awsVariantMigrationEnable(accountId);
+        if (migrationEnable) {
+            if (AwsConstants.AwsVariant.AWS_VARIANT.variant().value().equals(variant)) {
+                variant = AwsConstants.AwsVariant.AWS_NATIVE_VARIANT.variant().value();
+            }
+        }
+        return variant;
+    }
+
+    boolean awsVariantMigrationIsFeasible(Stack stack, String triggeredVariant) {
```

Comment:
nit: could you bring `is` to the front of the method name, so it would start with a verb as usual?
also could this be private please?